release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2373,8 +2423,12 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-18-known-issues_{context}"]
 == Known issues
 
+* oc-mirror plugin v2 currently returns an exit status of `0` (success) even when mirroring errors occur. As a result, customers should not rely on the exit status in automated workflows. Until this issue is resolved, users should manually check the `mirroring_errors_XXX_XXX.txt` file generated by `oc-mirror` for any errors. (link:https://issues.redhat.com/browse/OCPBUGS-49880[*OCPBUGS-49880*])
+
 * The DNF package manager included in {op-system-first} images cannot be used at runtime, because DNF relies on additional packages to access entitled nodes in a cluster that are under a Red Hat subscription. As a workaround, use the `rpm-ostree` command instead. (link:https://issues.redhat.com/browse/OCPBUGS-35247[*OCPBUGS-35247*])
 
+* When you used oc-mirror v2 Technology Preview in a mirror-to-mirror workflow, logs displayed catalog images twice. This could mislead users into thinking the catalog image was mirrored twice. In reality, the image was copied once to the destination registry and once to the oc-mirror local cache. This issue has been fixed to ensure logs accurately reflect the mirroring process. (link:https://issues.redhat.com/browse/OCPBUGS-41331[*OCPBUGS-41331*])
```

Comment:
```suggestion
* When you used oc-mirror v2 Technology Preview in a mirror-to-mirror workflow, logs displayed catalog images twice. This could mislead users into thinking that the catalog image was mirrored twice. In reality, the image was copied once to the destination registry and once to the `oc-mirror` local cache. This issue has been fixed to ensure that the logs accurately reflect the mirroring process. (link:https://issues.redhat.com/browse/OCPBUGS-41331[*OCPBUGS-41331*])
```

Reply:
Is this a bug fix or a KI? Seems like a bug fix.

Reply:
Added to the list for the writer to review.